x-pack/solutions/security/plugins/security_solution/public/flyout/ai_for_soc/components/attack_discovery_widget.test.tsx

```diff
@@ -7,19 +7,22 @@
 
 import { useLocation } from 'react-router-dom';
 import React from 'react';
-import { render, screen, fireEvent } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
We will have many merge conflicts here. I extracted part of `AttackDiscoveryWidget` into its own component, splitting up the tests: https://github.com/elastic/kibana/pull/219646

Reply:
Thanks for the heads up, fixing those conflicts now :)